coverage/src/BamViewChart.js

```diff
@@ -554,15 +559,15 @@ function createBamView(bamHeader, data, element, bamViewControlsElement, broker)
 
                 // Update the chromosome label to show the selected region
                 svg.selectAll('.chromosome-label')
-                    .text(`Chr ${orgChromosome}: ${Math.round(x0)} - ${Math.round(x1)} (${Math.round(x1 - x0)} bp)`);
+                    .text(`${bamHeaderMap[chromosomeEnd]}:${Math.round(x0)}-${Math.round(x1)} (${Math.round(x1 - x0)} bp)`);
```

Comment:
It was a bit confusing to me why you were using `chromosomeEnd` here, because I didn't realize you're using it as a unique identifier. That should work, but it's less explicit than using the actual sn.

Why do you need `bamHeaderMap`? Can't you just do `bamHeaderArray[chromosome - 1].sn`?

Reply:
Good question! Yes, that works. 

After giving it some thought, you reminded me that it might be better not to rely on the array index to determine the length or sequence number. This raises another question: Are the reference names always guaranteed to be in order in the header file? E.g (chr1 - chrY) or (1 - 24)

Reply:
Glad you picked up on that. I was planning to open another issue because I'm pretty sure that it's not guaranteed that X==23 and Y==24. In fact, you could have a header that only has chr5 for example, in which case the indices will totally be off. We need to account for that. It can be a separate PR if you want, or you can roll it into this one.

Reply:
I can fix it under this issue.